src/pages/index.js

```diff
@@ -8,38 +8,66 @@ import Section from "../components/Section.js";
 import PopupWithImage from "../components/PopupWithImage.js";
 import PopupWithForm from "../components/PopupWithForm.js";
 import UserInfo from "../components/UserInfo.js";
+import Api from "../components/api";
+import PopupWithConfirmation from "../components/PopupWithConfirmation";
 
 const addCardForm = document.querySelector("#add-card-form");
 const editProfileForm = document.querySelector("#edit-profile-form");
+const editAvatarForm = document.querySelector("#edit-avatar-form");
+
 const addNewCardButton = document.querySelector(".profile__add-button");
 const editProfileButton = document.querySelector(".profile__edit-button");
+const editAvatarButton = document.querySelector(".profile__image-edit-button");
+// const cardDeleteButton = document.querySelector(".card__delete-button");
 
 // create instances of classes
 
+const api = new Api({
+  baseUrl: "https://around-api.en.tripleten-services.com/v1",
+  headers: {
+    authorization: "538c4c21-92b1-4aec-a8cb-8245624705df",
+    "Content-Type": "application/json",
+  },
+});
+let cardSection;
+
+api.getInitialCards().then((res) => {
+  cardSection = new Section(
+    {
+      items: res,
+      renderer: (data) => {
+        const cardEl = createCard(data);
+        cardSection.addItem(cardEl);
+      },
+    },
+    selectors.cardSection
+  );
+  cardSection.renderItems();
+});
```

Comment:
You need to `catch` possible errors at the very end of any server request.

**Please, correct it everywhere**